test/integration/fixtures/blocks/core__form-input__deprecated-v2.serialized.html

```diff
@@ -0,0 +1,3 @@
+<!-- wp:form-input {"type":"checkbox","className":"wp-block-form-input"} -->
```

Comment:
Block deprecation works fine, but one thing that bothers me is that for some reason the block base class name is applied as the custom class name 🤔

![image](https://github.com/user-attachments/assets/6d7f78a3-3223-4162-a6fe-2d9f396b8b85)